uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java

```diff
@@ -40,14 +43,18 @@
 import com.microsoft.identity.client.ui.automation.logging.Logger;
 
 import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
 import java.util.Arrays;
 import java.util.List;
+import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 public class CommonUtils {
 
     private final static String TAG = CommonUtils.class.getSimpleName();
-    public final static long FIND_UI_ELEMENT_TIMEOUT = TimeUnit.SECONDS.toMillis(30);
+    public static long FIND_UI_ELEMENT_TIMEOUT = TimeUnit.SECONDS.toMillis(30);
```

Comment:
do we need this constant here `FIND_UI_ELEMENT_TIMEOUT`? I don't see it being used anywhere?